### PR TITLE
Notify users with sound when an ability has recharged

### PR DESCRIPTION
**Pull request by manu10000**
## Describe your changes
added a ping sound notification when abilities recharge. you're welcome jarif
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested my changes.

### CODE REVIEW

**Comment by Mykindos** [comment]
I'd like to do a setting for this, like drop protection & sidebar

**Comment by manu10000** [comment]
> I'd like to do a setting for this, like drop protection & sidebar

setting added

File: core/src/main/java/me/mykindos/betterpvp/core/client/repository/ClientListener.java [hunk 1]
```diff
@@ -258,6 +258,11 @@ private void checkUnsetProperties(Client client) {
         if(mapPlayerCaptionOptional.isEmpty()){
             client.saveProperty(ClientProperty.MAP_PLAYER_NAMES, false);
         }
+
+        Optional<Boolean> cooldownSoundOptional = client.getProperty(ClientProperty.MAP_PLAYER_NAMES);
```
Comment (Mykindos) [inline core/src/main/java/me/mykindos/betterpvp/core/client/repository/ClientListener.java:262]: hmmmmmm...

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/settings/menus/ChampionsSettingsMenu.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                .icon(ItemView.builder()
+                        .material(Material.NETHERITE_SWORD)
+                        .displayName(Component.text("Champions Settings", NamedTextColor.GREEN, TextDecoration.BOLD))
+                        .lore(Component.text("View generic settings related to champions", NamedTextColor.GRAY))
+                        .flag(ItemFlag.HIDE_ATTRIBUTES)
+                        .frameLore(true)
+                        .build())
+                .build();
+    }
+}
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/settings/menus/ChampionsSettingsMenu.java:56]: Cooldowns arent specific to champions, you can just put this in core setting smenu

**Review by Mykindos — CHANGES_REQUESTED** [review]

**Review by Mykindos — APPROVED** [review]

**Pull request merged**